cpp/src/IceSSL/OpenSSLEngine.cpp

```diff
@@ -44,61 +44,10 @@ mutex staticMutex;
 int instanceCount = 0;
 bool initOpenSSL = false;
 
-#if OPENSSL_VERSION_NUMBER < 0x10100000L || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
I think it doesn't make sense to support this older OpenSSL versions with 3.8. As per libressl I think is fine to remove this code we are not testing it.

Reply:
I agree!